script.js

```diff
@@ -1 +1,83 @@
-'use strict'
\ No newline at end of file
+'use strict'
+
+const title = document.querySelector('h1')
+const bodySelect = document.querySelector('body')
+const secretN = document.querySelector('.secret')
+const userInput = document.querySelector('.user--input')
+const btnCheck = document.querySelector('.btn--check')
+const btnReplay = document.querySelector('.btn--replay')
+const position = document.getElementById('position')
+const scoreSelect = document.getElementById('score')
+const highScoreSelect = document.getElementById('highScore')
+
+const player = {
+    score: '',
+    highscore: 0,
+    secret: 0,
+    
+    reset() {
+        secretNumber = Math.trunc(Math.random() * 20) + 1
+        score = 20
+    }
+}
```

Comment:
something is odd here and could be more simple.

`secretNumber` is variable pointing to `player.secret` and `player.reset()` is using `secretNumber`.

I think it's the perfect moment to use the famous `this`, it is ?

Reply:
je vois pas vraiment comment

Reply:
https://github.com/moonscomputerclub/devinombre/blob/master/script.js